sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -734,6 +734,42 @@ class SQLContext(@transient val sparkContext: SparkContext)
   def table(tableName: String): DataFrame =
     DataFrame(this, catalog.lookupRelation(Seq(tableName)))
 
+  /**
+   * Returns a [[DataFrame]] containing names of existing tables in the given database.
```

Comment:
the javadoc is actually wrong. this one should say "current database", and next one should say "given database"